dockerfile/cuda11.1.1.dockerfile

```diff
@@ -118,7 +118,7 @@ RUN cd /tmp && \
     rm -rf ./Linux mlc.tgz
 
 ENV PATH="${PATH}" \
-    LD_LIBRARY_PATH="/usr/local/lib:${LD_LIBRARY_PATH}" \
+    LD_LIBRARY_PATH="/usr/local/lib:/opt/hpcx/ompi/lib:${LD_LIBRARY_PATH}" \
```

Comment:
Not sure if we need to add more, such as `/opt/hpcx/ucx/lib`?

Reply:
Good point and updated!